app/src/main/java/com/framgia/ishipper/model/Invoice.java

```diff
@@ -249,6 +249,7 @@ public void setNumOfRecipient(String numOfRecipient) {
     }
 
     public ArrayList<InvoiceHistory> getHistories() {
+        if (mHistories == null) mHistories = new ArrayList<>();
```

Comment:
làm sao cần check thế này @@! Cần bảo toàn dữ liệu chứ, check gì thì check bên ngoài